tests/php/Dev/DeprecationTest.php

```diff
@@ -42,7 +45,9 @@ protected function setup(): void
 
     protected function tearDown(): void
     {
-        Deprecation::disable();
+        if (!$this->noticesWereEnabled) {
```

Comment:
When doing multi-pr tests using silverstripe/installer, I'm calling Deprecation::enable() in _config.php.  I've added funtionality to DeprecationTest to ensure we don't disable Deprecation notices after running DeprecatonTest